dask/bag/core.py

```diff
@@ -297,18 +299,33 @@ def remove(self, predicate):
                    for i in range(self.npartitions))
         return type(self)(merge(self.dask, dsk), name, self.npartitions)
 
-    def map_partitions(self, func):
+    def map_partitions(self, func, *args):
         """ Apply function to every partition within collection
 
         Note that this requires you to understand how dask.bag partitions your
         data and so is somewhat internal.
 
         >>> b.map_partitions(myfunc)  # doctest: +SKIP
+
+        Any additional arguments get passed to the function _after_ the
+        partition argument; argument values are first passed through
+        dask.imperative.to_task_dasks and any resulting dasks merged into the
+        result Bag dask:
+
+        >>> b.map_partitions(lambda part, freqs: part / freqs, b.frequencies())  # doctest: +SKIP
```

Comment:
This particular example, `part / freqs`, probably isn't ideal.  I'm unable to think of a situation where this would be valid.  Perhaps use something like `b.map_partitions(lambda part, total: [x / total for x in part], b.count())`?

Should this be extended to `map` as well?